src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react'
+import classnames from 'classnames'
+import { NavList } from '../../header/NavList/NavList'
+
+export type ExtendedNavLinksType = React.ReactNode[][]
+
+type FooterExtendedNavListProps = {
+  /* 
+    Multidimensional array of grouped nav links. Sub-arrays are column sections, first element is used as a heading.
+  */
+  nestedLinks: ExtendedNavLinksType
+}
+
+export const FooterExtendedNavList = ({
+  className,
+  nestedLinks,
+}: FooterExtendedNavListProps &
+  React.HTMLAttributes<HTMLElement>): React.ReactElement => {
+  const classes = classnames('grid-row grid-gap-4', className)
+
+  const [isMobile, setIsMobile] = React.useState(false)
+  const [sectionsOpenState, setSectionsOpenState] = useState<boolean[]>(
+    Array(nestedLinks.length).fill(false)
+  )
+
+  useEffect(() => {
+    const isClient = window && typeof window === 'object'
+    if (!isClient) return
+
+    function handleResize(): void {
+      const isMobileWidth = window.innerWidth < 480
+      setIsMobile(isMobileWidth)
```

Comment:
my only suggestion here would be to _only_ call `setIsMobile` if the value has changed, to avoid extraneous state updates